src/main/kotlin/no/nav/amt/aktivitetskort/domain/Aktivitetskort.kt

```diff
@@ -83,20 +83,23 @@ data class Aktivitetskort(
 		}
 
 		private fun deltakelseMengdeDetalj(deltaker: Deltaker): Detalj? {
-			val harDagerPerUke = deltaker.dagerPerUke?.let { it in 1..5 } == true
+			val harDagerPerUke = deltaker.dagerPerUke?.let { it in 1.0f..5.0f } == true
 			val harProsentStilling = deltaker.prosentStilling?.let { it in 1.0..100.0 } == true
 
 			val label = "Deltakelsesmengde"
 
 			fun fmtProsent(pct: Double) = "${DecimalFormat("#.#").format(pct)}%"
-			fun fmtDager(antall: Int) = "$antall ${if (antall == 1) "dag" else "dager"} i uka"
+			fun removeUnnessesaryDecimals(f: Float): String = if (f.rem(1) == 0.0f) f.toInt().toString() else f.toString()
```

Comment:
Bare nit, men `DecimalFormat("#.#")` fjerner også desimalen hvis den er 0. Også er det kanskje verdt å begrense antall desimaler til en eller to?

Reply:
Ah, den var fin, bruker den